modules/openapi-generator/src/main/resources/JavaSpring/pojo.mustache

```diff
@@ -37,10 +49,10 @@ public class {{classname}} {{#parent}}extends {{{.}}}{{/parent}}{{^parent}}{{#ha
   {{/isContainer}}
   {{^isContainer}}
   {{#isDate}}
-  @org.springframework.format.annotation.DateTimeFormat(iso = org.springframework.format.annotation.DateTimeFormat.ISO.DATE)
+  @DateTimeFormat(iso = DateTimeFormat.ISO.DATE)
   {{/isDate}}
   {{#isDateTime}}
-  @org.springframework.format.annotation.DateTimeFormat(iso = org.springframework.format.annotation.DateTimeFormat.ISO.DATE_TIME)
+  @DateTimeFormat(iso = DateTimeFormat.ISO.DATE_TIME)
   {{/isDateTime}}
   {{#openApiNullable}}
   private {{>nullableDataType}} {{name}}{{#isNullable}} = JsonNullable.undefined(){{/isNullable}}{{^isNullable}}{{#defaultValue}} = {{{.}}}{{/defaultValue}}{{/isNullable}};
```

Comment:
Out of curiosity, are there tests for a nullable field?

I had to fix this locally and haven't upstreamed the changes yet but we're also using a generator one version behind

Reply:
Yes, but those tests in the 3.0 spec: https://github.com/OpenAPITools/openapi-generator/blob/master/modules/openapi-generator/src/test/resources/3_0/petstore-with-fake-endpoints-models-for-testing.yaml#L1813

Spring samples (most if not all) are using 2.0 spec at the moment. We'll need to update these config to use the 3.0 spec instead.

Reply:
I'll look to update that in the coming weeks

Reply:
At least I introduced test configs for oas3 output format... We could update some of them to use 3.0 as input as well. But this could lead to even more test cases ...

oas2 -> oas2 -> ok
oas2 -> oas3 -> ok
oas3 -> oas2 -> do we even need this?
oas3 -> oas3 -> partially ok. This should be the fokus.